packages/babel-plugin-jest-deadlines/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "babel-plugin-jest-deadlines",
+  "version": "26.6.2",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/facebook/jest.git",
+    "directory": "packages/babel-plugin-jest-deadlines"
+  },
+  "engines": {
+    "node": "^10.13.0 || ^12.13.0 || ^14.15.0 || >=15.0.0"
+  },
+  "license": "MIT",
+  "main": "./build/index.js",
+  "types": "./build/index.d.ts",
+  "exports": {
+    ".": "./build/index.js",
+    "./package.json": "./package.json"
+  },
+  "dependencies": {
+    "@babel/template": "^7.3.3",
+    "@babel/types": "^7.3.3",
+    "@types/babel__core": "^7.0.0",
+    "@types/babel__traverse": "^7.0.6"
```

Comment:
unused?

Reply:
This started as a copy-paste of the hoist plugin. They are currently unused... however.

In the hoist plugin, these things are used in there to ensure that we have resolved the right globals, and to explicitly call jest's globals' expect, instead of just whatever variable named `expect` happens to be in scope, like my plugin does.

I wonder if we need to copy (share) the functionality from `hoist`?

Reply:
@jeysal thoughts?

Reply:
Yeah I think Jest is used enough that someone will have a shadowing `expect` that means a different thing and run into this, so we'll need to handle it. It should be relatively easy to insert a `const identifierGeneratedByBabel = expect` at top level and reference that instead

Reply:
I made it prepend `const generatedByBabelExpect = require("@jest/globals").expect;` and reference that, which should cover most usecases? I worry about writing it as `const generatedByBabelExpect = expect`; that's not going to work in some case, surely.

Reply:
Yeah that was written a bit hastily, I suppose that would collide with a `var expect` declared below or something. Reading the `expect` from the global this may work. But I think your solution is good.
@SimenB was there some bugs remaining when using certain things from `@jest/globals` that may be relevant here, or is it fine? Maybe I'm confusing it with the `globalThis.expect !== require('expect')` thing

Reply:
e.g. detox overrides `global.expect`: https://github.com/facebook/jest/pull/10484#issuecomment-703761828

not sure how much of an issue in practice that is.

It might make more sense to attach this only to the test runner (on `test`, or maybe even `jest`) rather than adding it to `expect`? Not sure